src/main/resources/assets/emi_loot/lang/ru_ru.json

```diff
@@ -0,0 +1,348 @@
+{
+  "emi.category.emi_loot.chest_loot": "Добыча из сундука",
+  "emi.category.emi_loot.block_drops": "Выпадает из блоков",
+  "emi.category.emi_loot.mob_drops": "Выпадает из мобов",
+  "emi.category.emi_loot.gameplay_drops": "Выпадает в игровом процессе",
+  "emi.category.emi_loot.archaeology_drops": "Добыча археологии",
+  "enchantment.emi_loot.random": "Случайно зачарованный",
+  "enchantment.emi_loot.random.desc": "К этому выпадению из доступного списка применяются случайно наложенные чары.",
+  "emi_loot.pickaxe.wood": "Необходима любая кирка",
+  "emi_loot.pickaxe.stone": "Необходима минимум каменная кирка",
+  "emi_loot.pickaxe.iron": "Необходимa минимум железная кирка",
+  "emi_loot.pickaxe.diamond": "Необходимa минимум алмазная кирка",
+  "emi_loot.pickaxe.netherite": "Необходимa минимум незеритовая кирка",
+  "emi_loot.axe.wood": "Необходим любой топор",
+  "emi_loot.axe.stone": "Необходим минимум каменный топор",
+  "emi_loot.axe.iron": "Необходим минимум железный топор",
+  "emi_loot.axe.diamond": "Необходим минимум алмазный топор",
+  "emi_loot.axe.netherite": "Необходим минимум незеритовый топор",
+  "emi_loot.shovel.wood": "Необходима любая лопата",
+  "emi_loot.shovel.stone": "Необходима минимум каменная лопата",
+  "emi_loot.shovel.iron": "Необходима минимум железная лопата",
+  "emi_loot.shovel.diamond": "Необходима минимум алмазная лопата",
+  "emi_loot.shovel.netherite": "Необходима минимум незеритовая лопата",
+  "emi_loot.hoe.wood": "Необходима любую мотыга",
+  "emi_loot.hoe.stone": "Необходимa минимум каменная мотыга",
+  "emi_loot.hoe.iron": "Необходимa минимум железная мотыга",
+  "emi_loot.hoe.diamond": "Необходимa минимум алмазная мотыга",
+  "emi_loot.hoe.netherite": "Необходимa минимум незеритовая мотыга",
+  "emi_loot.percent_chance": "%s%% шанс",
+  "emi_loot.percentage": "%s%%",
+  "emi_loot.rolls": "%s прокруток",
+  "emi_loot.rolls_visible": "%s раз",
+  "emi_loot.and": " & ",
+  "emi_loot.color_name": "%1$s %2$s",
+  "emi_loot.missing_key": "Отсутствующая или неправильно сформированная функция/состояние",
+  "emi_loot.missing_entity": "Неизвестная сущность",
```

Comment:
Сущность -> моб

Reply:
This applies to every use of "Сущность" in the lang file. You should use either "существо" or "моб". NOT "Сущность" as it has a completely different meaning on the technical side

Reply:
Agree, that "entity" is either "сущность" or "существо", but not "mob".
For "моб" there is a word "mob".
Moreover, for "существо" there is a word "creature".

However, it will be weird to see "block entity" as "блочный моб" or "блочное существо", because it's not a living entity.